tests/test_union.py

```diff
@@ -2,7 +2,7 @@
 import sys
 from dataclasses import dataclass
 from ipaddress import IPv4Address
-from typing import Dict, Generic, List, Optional, Tuple, TypeVar, Union
+from typing import Dict, Generic, List, Literal, Optional, Tuple, TypeVar, Union
```

Comment:
You can import `Literal` from `serde.compat`  👍 

https://github.com/yukinarit/pyserde/blob/master/serde/compat.py#L21-L26